index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio</title>
+    <!-- css link -->
+    <link rel="stylesheet" href="./assets/css/main.css" />
+  </head>
+  <body>
+    <!-- navigatin section -->
+    <nav id="navbar">
+      <div class="logo">
+        <a href="#">Eric Mahare</a>
+      </div>
+      <span class="nav_icon">
+        <img src="./assets/images/icons/hamburger.png" alt="" />
+      </span>
+    </nav>
+    <!-- end of navigation section -->
+    <!-- header section -->
+    <header id="header">
+      <h1 class="header_primary_text">Hey There. I’m Eric</h1>
+      <h4 class="header_supporting_text">I’am a Software Developer</h4>
+      <p class="header_description_text">
+        I can help you build a product , feature or website Look through some of
+        my work and experience! If you like what you see and have a project you
+        need coded, don’t hestiate to contact me.
+      </p>
+      <div class="social_media">
+        <ul>
+          <li><a href="#"><img src="./assets/images/icons/Vector.png" alt="github link"></a></li>
+          <li><a href="#"><img src="./assets/images/icons/Vector-1.png" alt="github link"></a></li>
+          <li><a href="#"><img src="./assets/images/icons/Vector-2.png" alt="github link"></a></li>
+          <li><a href="#"><img src="./assets/images/icons/Vector-3.png" alt="github link"></a></li>
+          <li><a href="#"><img src="./assets/images/icons/Vector-4.png" alt="github link"></a></li>
+        </ul>
+      </div>
+      <aside class="illustration">
+        <img src="./assets/images/Illustration.png" alt="">
+      </aside>
+    </header>
+    <!-- end of header section -->
```

Comment:
As mentioned above kindly let's update the usage of HTML tags as of the project requirement this section is the `headline` section so let's replace the `header` tag with the `section` tag the content inside should have to remain the same. This section has to be right after the `header` section. Here is the screenshot from the project requirement: 
<img width="711" alt="image" src="https://user-images.githubusercontent.com/34578631/158764540-5d898053-54cd-4f89-8791-cc59a4b2b17a.png">